src/plugins/discourse/mirror.js

```diff
@@ -473,6 +473,16 @@ export class Mirror implements DiscourseData {
         }
         offset += likeActions.length;
       }
+    };
+    for (const user of this.users()) {
+      try {
+        await addUserLikes(user);
+      } catch (e) {
```

Comment:
I believe this is far too permissive of the problem we're trying to be robust to. For example, bug in the SQL query would cause ignores here, while I feel they should crash.